tools/db_crashtest.py

```diff
@@ -179,6 +179,7 @@
     "wal_compression": "none",
     "verify_sst_unique_id_in_manifest": 1,  # always do unique_id verification
     "secondary_cache_uri": "",
+    "test_secondary": 1,
```

Comment:
It looks like we use secondary to implement `continuous_verification_interval > 0`. If so can `continuous_verification_interval > 0` imply we open a secondary without a new option for it?

Reply:
Good question.
The reason I used a different command line flag is that, even if `continuous_verification_internal > 0`, it does not logically indicate that we will use secondary instance for verification. Theoretically, we can use the original db pointer for the same purpose.